src/app/components/tree-view/index.html

```diff
@@ -49,23 +53,23 @@ <h3 class="sky-text-success">Use when</h3>
       <h3>Selection</h3>
```

Comment:
Should we call out read-only mode and toolbar somewhere in the Options section of the Design docs? It seems like we should since these are valid options of the components we built. I especially think this is the case for the read-only mode because we call it out in the Use when section and then in the Options section, we start right in on the selection modes. And we highlight both items in the demo.

Reply:
Yeah I will add something, probably just a text description since I don't know how to illustrate things that aren't allowed ;).

Reply:
Sounds good.